tools/codegen/cmd/featuregate-test-analyzer.go

```diff
@@ -363,6 +365,21 @@ var (
 		//},
 	}
 
+	// These are only checked if the feature gate is platform specific
```

Comment:
Why shouldn't we always checks and if a featuregate is targeted at a particular variant, then restrict to just that one?